ai-service/tests/test_llm_and_search.py

```
@@ -1,76 +1,79 @@
+# -------------- LLM Chat Tests --------------
 
-# def test_llm_chat_basic_response(monkeypatch: pytest.MonkeyPatch):
-#     monkeypatch.setattr(
-#         "ai_service.ollama_client.chat_with_ollama",
-#         lambda prompt: "Mocked LLM response for: " + prompt,  # type: ignore
-#     )
-#     prompt = "Code context:\ndef add(a, b): return a + b\nQuestion: How does the sum work?\nExplain."
-#     response = ollama_client.chat_with_ollama(prompt)
-#     assert response.startswith("Mocked LLM response for:")
 
+def test_llm_chat_basic_response(monkeypatch: pytest.MonkeyPatch):
```

Ruff (0.12.2)

Missing return type annotation for public function `test_llm_chat_basic_response`

Add return type annotation: `None`

(ANN201)